corehq/apps/reports/filters/users.py

```diff
@@ -26,6 +27,9 @@
     BaseSingleOptionFilter,
 )
 
+#TODO: replace with common code
+mark_safe_lazy = lazy(mark_safe, str)
```

Comment:
Would `corehq/util/markup.py` be a good home for this? (Is that kinda what you mean by "replace with common code"?)

Reply:
That would be a good candidate. I believe I had two branches going simultaneously, where one had created a separate module for this including tests. While this branch was also in flight, I left the comment to switch these over to use that module, when the other branch is merged. I'll create a reminder for myself to remember to do that, but with these changes due on the 12th, I likely won't have a chance to address them prior